### PR TITLE
Nessie CLI cleanup

### PR DESCRIPTION
**Pull request by nastra**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/2017)

### CODE REVIEW

File: site/docs/tools/cli.md [hunk 1]
```diff
@@ -158,38 +187,64 @@ Usage: nessie log [OPTIONS] [REVISION_RANGE] [PATHS]...
...
 ``` bash
 $ nessie merge --help
 ``` 
 
 ``` bash
-Usage: nessie merge [OPTIONS] [MERGE_BRANCH]
+Usage: nessie merge [OPTIONS] [FROM_BRANCH]
 
-  Merge BRANCH into current branch. BRANCH can be a hash or branch.
+  Merge FROM_BRANCH into current branch. FROM_BRANCH can be a hash or branch.
```
Comment (snazy) [inline site/docs/tools/cli.md:237]: "Current" is confusing here. It either merged into the default branch or the explicitly specified branch. Maybe just use "another"?

File: site/docs/tools/cli.md [hunk 2]
```diff
@@ -25,19 +26,21 @@ Usage: nessie [OPTIONS] COMMAND [ARGS]...
...
+  --help             Show this message and exit.
 
 Commands:
   branch       Branch operations.
   cherry-pick  Transplant HASHES onto current branch.
   config       Set and view config.
   contents     Contents operations.
   log          Show commit log.
-  merge        Merge BRANCH into current branch.
+  merge        Merge FROM_BRANCH into current branch.
```
Comment (snazy) [inline site/docs/tools/cli.md:43]: "Current" is confusing here. It either merged into the default branch or the explicitly specified branch. Maybe just use "another"?

File: site/docs/tools/cli.md [hunk 3]
```diff
@@ -25,19 +26,21 @@ Usage: nessie [OPTIONS] COMMAND [ARGS]...
...
+  --endpoint TEXT    Optional endpoint, if different from config file.
+  --auth-token TEXT  Optional bearer auth token, if different from config
+                     file.
   --version
-  --help           Show this message and exit.
+  --help             Show this message and exit.
 
 Commands:
   branch       Branch operations.
   cherry-pick  Transplant HASHES onto current branch.
```
Comment (snazy) [inline site/docs/tools/cli.md:39]: "Current" is confusing here. It either merged into the default branch or the explicitly specified branch. Maybe just use "another"?

File: site/docs/tools/cli.md [hunk 4]
```diff
@@ -91,34 +106,34 @@ Usage: nessie cherry-pick [OPTIONS] [HASHES]...
   Transplant HASHES onto current branch.
```
Comment (snazy) [inline site/docs/tools/cli.md:106]: "Current" is confusing here. It either merged into the default branch or the explicitly specified branch. Maybe just use "another"?

File: site/docs/tools/cli.md [hunk 5]
```diff
@@ -134,19 +149,33 @@ Usage: nessie contents [OPTIONS] [KEY]...
...
+  -c, --condition TEXT            Conditional Hash. Only perform the action if
+                                  the branch currently points to the hash
+                                  specified by this option.
+  -r, --ref TEXT                  branch to list from. If not supplied the
+                                  default branch from config is used
+  -m, --message TEXT              commit message
+  -t, --type TEXT                 entity types to filter on, if no entity
+                                  types are passed then all types are returned
+  --query, --query-expression TEXT
+                                  Allows advanced filtering using the Common
```
Comment (snazy) [inline site/docs/tools/cli.md:164]: The CEL references are good - I like it. It's just hard to read here with the small column size. Can we widen the column at least and add some newlines to the message?

File: site/docs/tools/cli.md [hunk 6]
```diff
@@ -158,38 +187,64 @@ Usage: nessie log [OPTIONS] [REVISION_RANGE] [PATHS]...
...
+                                  date
+  --author TEXT                   Limit commits to a specific author (this is
+                                  the original committer). Supports specifying
+                                  multiple authors to filter by.
+  --committer TEXT                Limit commits to a specific committer (this
+                                  is the logged in user/account who performed
+                                  the commit). Supports specifying multiple
+                                  committers to filter by.
+  --query, --query-expression TEXT
+                                  Allows advanced filtering using the Common
```
Comment (snazy) [inline site/docs/tools/cli.md:213]: The CEL references are good - I like it. It's just hard to read here with the small column size. Can we widen the column at least and add some newlines to the message? Or maybe a section/note about CEL on the website and link to that page from the pynesse CLI help?

File: site/docs/tools/cli.md [hunk 7]
```diff
@@ -158,38 +187,64 @@ Usage: nessie log [OPTIONS] [REVISION_RANGE] [PATHS]...
...
-  -n, --number INTEGER    number of log entries to return
-  --since, --after TEXT   Commits more recent than specific date
-  --until, --before TEXT  Commits older than specific date
-  --author, --committer   limit commits to specific committer
-  --help                  Show this message and exit.
+  -r, --ref TEXT                  branch to list from. If not supplied the
+                                  default branch from config is used
+  -n, --number INTEGER            number of log entries to return
+  --since, --after TEXT           Only include commits newer than specific
+                                  date
```
Comment (snazy) [inline site/docs/tools/cli.md:202]: Mind adding an example date format? Or maybe a section/note about date formats on the website and link to that page from the pynesse CLI help?

**Review by snazy — COMMENTED** [review]
Like it!
Couple of comments though

**Comment by nastra** [comment]
@snazy I'll update the cmd line descriptions based on your comments but I came to the conclusion that keeping CLI docs manually in sync with the website is terrible and we should just refer to https://nessie.readthedocs.io/en/latest/cli.html here, which I already did in #2019

File: site/docs/tools/cli.md (hunk 1 above)
Comment (nastra) [reply site/docs/tools/cli.md:237]: I actually like `Merge FROM_BRANCH into default branch or the explicitly specified branch` as this should make it quite clear what's meant

File: site/docs/tools/cli.md (hunk 3 above)
Comment (nastra) [reply site/docs/tools/cli.md:39]: `Transplant HASHES onto default branch or the explicitly specified branch` sounds pretty good and self-explaining

Comment (nastra) [reply site/docs/tools/cli.md:39]:
```
  cherry-pick  Transplant HASHES onto default branch or the explicitly...
  config       Set and view config.
  contents     Contents operations.
  log          Show commit log.
  merge        Merge FROM_BRANCH into default branch or the explicitly...
```
I guess we'll use `another` then as the text is being shortened :(

**Comment by snazy** [comment]
Can we somehow auto-generate the page that's on readthedocs for our site when the files in `python/docs` are generated?

File: site/docs/tools/cli.md (hunk 7 above)
Comment (nastra) [reply site/docs/tools/cli.md]: good idea, done

**Comment by nastra** [comment]
> Can we somehow auto-generate the page that's on readthedocs for our site when the files in `python/docs` are generated?

I was thinking about that as well. I'll take a look at that outside of this PR.

File: python/pynessie/cli.py [hunk 8]
```diff
@@ -270,7 +270,7 @@ def log(  # noqa: C901
 
 def _format_log_result(x: CommitMeta) -> str:
     result = click.style("commit {}\n".format(x.hash_), fg="yellow")
-    result += click.style("Author: {} <{}>\n".format(x.committer, x.email))
+    result += click.style("Author: {}\n".format(x.committer))
```
Comment (snazy) [inline python/pynessie/cli.py:273]: Oh, shouldn't this be `x.author`?

FYI: git doesn't show the committer, even in the "very verbose commit view mode".

Comment (nastra) [reply python/pynessie/cli.py]: good catch, yes this should be `x.author`

**Review by snazy — APPROVED** [review]

**Pull request merged**